bouncer/src/repo/manipulator/modelconvertor/import/odaHelper/geometry_collector.cpp

```diff
@@ -79,94 +81,203 @@ repo::core::model::RepoNodeSet repo::manipulator::modelconvertor::odaHelper::Geo
 	return metaNodes;
 }
 
-mesh_data_t GeometryCollector::createMeshEntry() {
+mesh_data_t GeometryCollector::createMeshEntry(uint32_t format) {
 	mesh_data_t entry;
 	entry.matIdx = currMat;
 	entry.name = nextMeshName;
 	entry.groupName = nextGroupName;
 	entry.layerName = nextLayer.empty() ? "UnknownLayer" : nextLayer;
-
+	entry.format = format;
 	return entry;
 }
 
-void  GeometryCollector::startMeshEntry() {
-	nextMeshName = nextMeshName.empty() ? std::to_string(std::time(0)) : nextMeshName;
+void GeometryCollector::startMeshEntry() {
+	nextMeshName = nextMeshName.empty() ? boost::lexical_cast<std::string>(uuidGenerator()) : nextMeshName;
```

Comment:
we have a wrapper around boost uuid, might be better to use that instead? 

then you can just do 
```suggestion
	nextMeshName = nextMeshName.empty() ? RepoUUID::createUUID().toString() : nextMeshName;
```